### ISSUE TITLE
Data Link - "Open in new tab" button status mismatch

### ISSUE THREAD

**Issue by mac3000** [grafana/grafana#25445]
**What happened**:
When adding Data Links to Graph, the "Open in new tab" button in the frame will always show "Off", regardless if you changed it to before.

**What you expected to happen**:
Button should reflect the actual status

**How to reproduce it (as minimally and precisely as possible)**:
1. Add a data link to a graph, and select to open in new Window. Apply, Save and confirm it works as expected (you can also Refresh page)
2. Go back to edit the Link, it will show Button in "Off" setting. If you change to "On" and test, it will open in same Window.

Basically, that button will always show "Off", regardless of what you changed it to before.

Grafana version: Grafana v7.0.0 (aee1438ff2)

Let me know if you want or need more details.

Thanks,
Jacob

**Comment by torkelo** [comment]
thanks for reporting this

**Comment by tskarhed** [comment]
DataLinkEditor could be refactored using the Form component instead. Should probably do that. I also noticed that it is in grafana-ui, which feels a little bit weird.
